mypy/types.py

```diff
@@ -144,6 +144,97 @@ def deserialize(cls, data: JsonDict) -> 'Type':
         raise NotImplementedError('Cannot deserialize {} instance'.format(cls.__name__))
 
 
+class TypeAliasType(Type):
+    """
+    A type alias to another type.
```

Comment:
Style nit: move the first line to the same line with `"""` (`"""A type alias ...`).